### DISCUSSION TITLE
Gephi forum doesn't work

### DISCUSSION THREAD

**Discussion by ilkant** [gephi/gephi#2782] [Q&A] [answered]
Gephi forum https://forum-gephi.org/ shows errors:

[phpBB Debug] PHP Warning: in file [ROOT]/phpbb/session.php on line 583: sizeof(): Parameter must be an array or an object that implements Countable
[phpBB Debug] PHP Warning: in file [ROOT]/phpbb/session.php on line 639: sizeof(): Parameter must be an array or an object that implements Countable
[phpBB Debug] PHP Warning: in file [ROOT]/includes/functions.php on line 4516: Cannot modify header information - headers already sent by (output started at [ROOT]/includes/functions.php:3262)
[phpBB Debug] PHP Warning: in file [ROOT]/includes/functions.php on line 4516: Cannot modify header information - headers already sent by (output started at [ROOT]/includes/functions.php:3262)
[phpBB Debug] PHP Warning: in file [ROOT]/includes/functions.php on line 4516: Cannot modify header information - headers already sent by (output started at [ROOT]/includes/functions.php:3262)

And when you log in to you account, it gives more error messages and does not show anything. Please, if you are maintaining that forum, check it and fix the errors.

**Comment by mbastian** [comment, marked as answer]
Hi @ilkant the Gephi Forum is no longer maintained. It's still out there as an archive. We recommend reaching out to the community either here, or on the Facebook Group.
